modules/tinymce/src/core/test/ts/browser/fmt/ExpandRangeTest.ts

```diff
@@ -226,6 +226,21 @@ describe('browser.tinymce.core.fmt.ExpandRangeTest', () => {
       const rng = expandRng(editor, [ 0, 0, 0, 0, 0 ], 0, [ 0, 0, 0, 0, 0 ], 1, blockFormat, false);
       assertRange(editor, rng, [ 0, 0, 0 ], 0, [ 0, 0, 0 ], 1);
     });
+
+    it('TINY-10154: should expand over the nested summary content instead of parent details body element', () => {
```

Comment:
You also talked about table cases etc should we add some more tests here to cover all of them in case we ever refactor that code we want it to work for those cases as well.

Reply:
done